userspace/libsinsp/container_engine/cri.cpp

```diff
@@ -306,7 +306,8 @@ bool cri::resolve(sinsp_threadinfo *tinfo, bool query_os_for_missing_info)
 
 		if(!m_async_source)
 		{
-			auto async_source = new cri_async_source(cache, m_cri.get(), s_cri_timeout);
+			uint64_t max_wait_ms = 10000;
+			auto async_source = new cri_async_source(cache, m_cri.get(), max_wait_ms);
```

Comment:
Why aren't we using `s_cri_timeout` anymore here?

Reply:
I talk about it a little bit in the checkin comments. This param is really about the lifetime of a given value in the async_key_value_source values map and not the timeout for grpc queries when using cri. That's why I switched to its own value.

The cri timeout is still used when doing those grpc queries (see https://github.com/falcosecurity/libs/blob/master/userspace/libsinsp/cri.cpp#L54 and friends)

Reply:
Thanks; makes sense!